app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
```

Comment:
- [ ] Good job with the Project Team. But, please remember the views in device for the registration and login have been modified to match the wireframes and their styling. your design is a bit off compared to the wireframe. 

### Your layout 
![Screen Shot 2023-09-09 at 11 10 14 AM](https://github.com/ibugithub/R-Blog/assets/69077061/8703cd4e-8865-40b1-a746-c0f3a4437698)

### Wirefrane
![blog_login](https://github.com/ibugithub/R-Blog/assets/69077061/6775f807-35d7-41d8-b631-9e1ee1544538)

**Please update the layout to match the wireframe**